src/components/FilterBarInline/FilterBarInline.mdx

```diff
@@ -0,0 +1,349 @@
+will be set to `"full"`.
+
+**Note:** `CheckboxGroup`, `MultiSelectGroup`, and `RadioGroup` `layout` prop will
+match the value of the `layout` set for the `FilterBarInline` component.
+
+<Canvas of={FilterBarInlineStories.LayoutPatterns} />
+
+## UI Containers
+
+The `FilterBarInline` component will often be rendered within a UI container with
+`background-color` and `padding` styles applied. This treatment can easily be
+accomplished using the `Box` component with inline styles.
+
+If a consuming app supports dark mode, styles for both light and dark mode will
+be required for the UI container element.
+
+**Note:** Future DS updates may provide a standalone component to implement a
+UI container similar to what is used in these example.
+
+### Row Layout
+
+<Canvas of={FilterBarInlineStories.UIContainers} />
+
+<Source
+  code={`
+/* LIGHT MODE ONLY */
+// Add container for row layout
+<Box bg="ui.bg.default" p="inset.wide">
+  <FilterBarInline
+    layout="row"
+    renderChildren={() => {
+      // Filter components...
+    }}
+    // ...
+  />
+</Box>
+`}
+  language="jsx"
+/>
+
+<Source
+  code={`
+/* DARK MODE SUPPORT */
+// Import useColorModeValue hook
+import { useColorModeValue } from "@nypl/design-system-react-components";
+// ...
+// Use hook to set values for light and dark mode
+const containerBgColor = useColorModeValue(
+  "ui.bg.default",
+  "dark.ui.bg.default"
+);
+// ...
+// Add container for row layout
+<Box bg={containerBgColor} p="inset.wide">
+  <FilterBarInline
+    layout="row"
+    renderChildren={() => {
+      // Filter components...
+    }}
+    // ...
+  />
+</Box>
+; `}
+  language="jsx"
+/>
+
+### Column Layout
+
+<Canvas of={FilterBarInlineStories.ColumnLayout} />
+
+<Source
+  code={`
+/* LIGHT MODE ONLY */
+// Add container for column layout
+<Box bg="ui.bg.default" p="inset.default">
+  <FilterBarInline
+    layout="column"
+    renderChildren={() => {
+      // Filter components...
+    }}
+    // ...
+  />
+</Box>
+`}
+  language="jsx"
+/>
+
+<Source
+  code={`
+/* DARK MODE SUPPORT */
+// ...
+// Import useColorModeValue hook
+import { useColorModeValue } from "@nypl/design-system-react-components";
+// ...
+// Use hook to set values for light and dark mode
+const containerBgColor = useColorModeValue(
+  "ui.bg.default",
+  "dark.ui.bg.default"
+);
+// ...
+// Add container for column layout
+<Box bg={containerBgColor} p="inset.default">
+  <FilterBarInline
+    layout="column"
+    renderChildren={() => {
+      // Filter components...
+    }}
+    // ...
+  />
+</Box>;
+`}
+  language="jsx"
+/>
+
+## Passing Children
+
+<Canvas of={FilterBarInlineStories.MultipleChildren} />
+
+In order to control the styling and props passed to children form elements,
+the `FilterBarInline` component accepts `CheckboxGroup`, `MultiSelectGroup`,
+`RadioGroup`, `Checkbox`, `DatePicker`, `MultiSelect`, `Radio`, `Select`,
+`Slider`, `TextInput`, and `Toggle` children through its `renderChildren` render
```

Comment:
You give really good full examples below, but I think a short snippet to explain the shape of this will be helpful. Something like
```
<FilterBarInline
  id="..."
  heading="..."
  renderChildren={({ isBlockElement, layout, width }) => {
    return (...)
  }}
/>
```